src/dsl/matchers.ts

```diff
@@ -253,6 +253,13 @@ export function boolean(value: boolean = true) {
   return somethingLike<boolean>(value)
 }
 
+/**
+ * String Matcher.
+ */
+export function string(value: string = '') {
+  return somethingLike<string>(value);
```

Comment:
Can you please use a non-empty string here? You can choose something interesting, otherwise something like `"example"` will do.

Reply:
Done